command/operator_migrate.go

```diff
@@ -135,6 +144,11 @@ func (c *OperatorMigrateCommand) Run(args []string) int {
 	}
 	c.logger = logging.NewVaultLogger(log.LevelFromString(c.flagLogLevel))
 
+	if (c.flagStart != "") && c.flagParallel != 1 {
```

Comment:
Please could we add some sanity check on the `c.flagParallel` value before this to ensure it's `>= 1`? You could modify this `if` statement to check `c.flagParallel > 1` after that.